calour/io.py

```diff
@@ -286,8 +261,48 @@ def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_m
     return exp
 
 
+def _read_metadata(ids, f, kwargs):
+    '''read metadata table
+
+    Parameters
+    ----------
+    ids : list like of str
+        ids from data table
```

Comment:
can you add explanation what ids is used for (i.e. reorder and keep only data about ids)

Reply:
this explanation shouldn't be here? remember this is private function